### PR TITLE
add histo to calculate the average multiplicity in the transverse region

### PR DESCRIPTION
**Pull request by gluparel**
add histo to calculate the average multiplicity in the transverse region

### CODE REVIEW

File: PWGHF/vertexingHF/AliAnalysisTaskSEDvsRT.h [hunk 1]
```diff
@@ -91,7 +91,7 @@ class AliAnalysisTaskSEDvsRT : public AliAnalysisTaskSE
    TH1F *fGlobalRT;        //!<! hist for RT distribution without D-meson selection
    TH1F *fHistPtLead;      //!<! hist for pT distribution of leading track
    TH3F *fRTvsZvtxvsMult;        //!<! distribution of RT as function of z of primary vertex and tracklet multiplicity
-   
+   TH1F *fNChargedInTrans; //!<! hist for NChargedDistribution
```
Comment (fgrosa) [inline PWGHF/vertexingHF/AliAnalysisTaskSEDvsRT.h:94]: you missed the update of the `ClassDef`

**Review by fgrosa — COMMENTED** [review]
Ciao @gluparel as soon as you update the `ClassDef` I approve it, thanks a lot!

File: PWGHF/vertexingHF/AliAnalysisTaskSEDvsRT.h (hunk 1 above)
Comment (gluparel) [reply PWGHF/vertexingHF/AliAnalysisTaskSEDvsRT.h:94]: Ciao @fgrosa, sorry I always forget this...

**Comment by fgrosa** [comment]
+1

**Pull request merged**